vignettes/visits_periods.Rmd

```diff
@@ -94,51 +94,193 @@ derive_vars_joined(
 
 If periods, subperiods, or phases are used, the corresponding variables
 have to be consistent across all datasets. This can be achieved by
-defining the periods, subperiods, or phases once and then use this
+defining the periods, subperiods, or phases once and then using this
 definition for all datasets. The definition can be stored in ADSL or in
 a separate dataset. In the following examples, this separate dataset is
-called period reference dataset.
+called period/phase reference dataset depending what data it contains.
 
-## Period Reference Dataset {#reference}
+## A Note on Study Specific Code
 
-The period reference dataset contains one observation per subject and
-period, subperiod, or phase. For example:
+The sections below showcase the available tools in admiral to work with period, 
+subperiod and phase variables. However, at some point study specific code will 
+always be required. There are two options:
 
-```{r echo=FALSE}
-phase_ref <- tribble(
-  ~USUBJID, ~APHASEN, ~PHSDT,       ~PHEDT,       ~APHASE,
-  "1",             1, "2021-01-04", "2021-02-06", "TREATMENT",
-  "1",             2, "2021-02-07", "2021-03-07", "FUP",
-  "2",             1, "2021-02-02", "2021-03-02", "TREATMENT"
-) %>%
-  mutate(
-    STUDYID = "xyz",
-    APHASEN = as.integer(APHASEN),
-    across(matches("PH[ES]DT"), ymd)
-  ) %>%
-  select(STUDYID, everything())
+-   Study specific code is used to first derive the variables `PxxSwSDT` and
+`PxxSwEDT` in ADSL. Then `create_period_dataset()` and
+`derive_vars_joined()` can be used to derive period/subperiod
+variables like `ASPER` or `ASPRSDT` in BDS and OCCDS datasets. (See
+[example](#adsl_example).)
 
-phase_ref
-```
+-   Study specific code is used to derive a dataset with one
+observation per patient, period, and subperiod (see [period
+reference dataset](#reference)). Then `derive_vars_period()` can be
+used to derive `PxxSwSDT` and `PxxSwEDT` in ADSL and
+`derive_vars_joined()` can be used to derive period/subperiod
+variables like `ASPER` or `ASPRSDT` in BDS and OCCDS datasets.
+
+It depends on the specific definition of the periods/subperiods which
+option works best. If the definition is based on other ADSL variables,
+the first option would work best. If the definition is based on
+vertically structured data like exposure data (EX dataset), the second
+option should be used. This vignette contains examples for both workflows.
+
+
+## The Period/Phase Reference Dataset {#reference}
+
+The period/phase reference dataset should be created according to the design and ADaM needs of the study in question. It should contain one observation per subject and
```

Comment:
Could this paragraph be moved after the next paragraph. could then just say "See below for an example dataset".